src/workers/codegen.ts

```diff
@@ -72,7 +72,12 @@ export const setupCodegenWorker: SetupCodegenWorkerFn = ({
       codegenOptions.pluginMap['typescript'] = require('@graphql-codegen/typescript');
       codegenOptions.plugins.push({ typescript: DEFAULT_TYPESCRIPT_CONFIG });
       codegenOptions.pluginMap['typescriptOperations'] = require('@graphql-codegen/typescript-operations');
-      codegenOptions.plugins.push({ typescriptOperations: DEFAULT_TYPESCRIPT_CONFIG });
+      codegenOptions.plugins.push({
+        typescriptOperations: {
+          ...DEFAULT_TYPESCRIPT_CONFIG,
+          exportFragmentSpreadSubTypes: true
```

Comment:
It can be merged into `DEFAULT_TYPESCRIOT_CONFIG`. I will do and merge

Thank you!

Reply:
I was avoiding passing the nonexistent setting to the Typescript plugin, because I wasn't sure if it would cause a warning or something. Otherwise, sounds good!

Reply:
Make sense. Could you add the same option to flow-operations plugin?

Reply:
Added. I don't use Flow so I can't confirm that the setting works for `graphql-codegen-flow-operations`, but it is documented here: https://graphql-code-generator.com/docs/plugins/flow-operations